packages/react-components/react-combobox/src/utils/Selection.types.ts

```diff
@@ -14,11 +14,11 @@ export type SelectionProps = {
   multiselect?: boolean;
 
   /* Callback when an option is selected */
-  onSelect?(event: SelectionEvents, data: OnSelectData): void;
+  onOptionSelect?: (event: SelectionEvents, data: OptionSelectData) => void;
```

Comment:
Typically these types are `On....`
```suggestion
  onOptionSelect?: (event: SelectionEvents, data: OnOptionSelectData) => void;
```

Reply:
Huh, I actually changed it away from `On*` because when I looked around, it seemed like the pattern was to not include `On` in the data type 😅

e.g. `MenuOpenChangeData`, `DialogOpenChangeData`, `InputOnChangeData` (granted, that sort-of has "on"), `SelectTabData`, probably others I didn't check 😅

Reply:
oh interesting, there are a few more controls that follow the pattern of `InputOnChangeData`. Is that what you were suggesting? I assumed you meant `OnOptionSelectData` 😅.

`OptionOnSelectData` makes sense to me, since a few other form controls follow that pattern. Either way, it seems unfortunate that we aren't super consistent across all components, but I don't think we can change all of them now 😬

Reply:
> Is that what you were suggesting?

Yeah, sorry I remembered the word order wrong 🤦

All of the input components should use the "on" naming after the [coherence pass](https://github.com/microsoft/fluentui/issues/21809#issuecomment-1050135988).